### ISSUE TITLE
client.getMe() return undefined

### ISSUE THREAD

**Issue by itzharDev** [smashah/sulla#168]
client.getMe() return undefined

maybe related to https://github.com/mukulhase/WebWhatsapp-Wrapper/issues/503

**Comment by joweste** [comment]
Hi, I create this new function.
It works to me. Maybe to you.
If the mantainer whants, he can include it into source.
It´s my contribution

```
window.WAPI.CloneObj = (obj) => JSON.parse(JSON.stringify(obj));

window.WAPI.getMe = function (done) {
    let found = WAPI._serializeContactObj(window.Store.Contact.get(window.Store.Conn.me));
    let newjsonobj = WAPI.CloneObj(found);
    if (done !== undefined) done(newjsonobj);
    return newjsonobj;
};

```
Regards, Luiz Alves

**Comment by smashah** [comment]
Please follow the template properly next time. This getMe function is different from the referenced one.

![release](https://user-images.githubusercontent.com/5903821/75934031-5a73a600-5e73-11ea-8638-826971863dde.png)

**Comment by joweste** [comment]
Sorry, but exists a template to fix a function?
I am asking because I don´t know.

**Comment by itzharDev** [comment]
just upload fix on 1.8.8

**Comment by smashah** [comment]
Tested and working in v.1.8.10
